selfdrive/ui/qt/spinner.cc

```diff
@@ -19,8 +19,8 @@ TrackWidget::TrackWidget(QWidget *parent) : QWidget(parent) {
   setFixedSize(spinner_size);
 
   // pre-compute all the track imgs. make this a gif instead?
-  QPixmap comma_img = QPixmap("../assets/img_spinner_comma.png").scaled(spinner_size, Qt::KeepAspectRatio, Qt::SmoothTransformation);
-  QPixmap track_img = QPixmap("../assets/img_spinner_track.png").scaled(spinner_size, Qt::KeepAspectRatio, Qt::SmoothTransformation);
+  QPixmap comma_img = scaledPixmap("../assets/img_spinner_comma.png", spinner_size);
```

Comment:
this would be for its own PR, but what do you think of putting these utils in their own namespace? it's unclear where this scaledPixmap comes from right now.